src/componentLibrary/theme/theme.interlay.css

```diff
@@ -19,5 +19,8 @@
   --colors-text-tertiary: var(--colors-neutral-light-grey);
   --colors-modal-bg: var(--colors-neutral-white);
   --colors-icon-default: var(--colors-neutral-white);
+  --colors-table-bg: var(--colors-neutral-white);
+  --colors-table-text: var(--colors-neutral-black);
+  --colors-new-vaults-table-highlight: var(--colors-light-blue);
```

Comment:
This could be made less specific—Jay will standardise things like this, so `colors-table-row-highlight` or similar would be ok,